src/rpc/scgi_task.cc

```diff
@@ -1,14 +1,16 @@
 #include "config.h"
 
+#include "rpc/parse_commands.h"
```

Comment:
Do not use rpc commands here.

Instead the min size setting should be a member variable of SCgiTask.

Reply:
Moved to a static member var